### PR TITLE
Closes #81 - This adds `Small News Article Thumbnails`

### LINKED ISSUE #81: Create new BB: Small News Article Thumbnails (3-up)
This is a Building Block for our upcoming News/Magazine Kit. For anyone who wants to contribute some building blocks to the site, feel free to comment below to let us know you're taking this one on!

**Description/Interaction:** Simple article links on the homepage of a news site. Should include at least: an image, a tag, and a header. Can also include author and publish date if you want. Have fun with it!

**Example:**
screen shot 2017-04-14 at 4 44 38 pm

### PR DESCRIPTION
**Pull request by IamManchanda**
Closes #81 Poke @canterberrie for review!

3-up-news-cards

### CODE REVIEW

**Comment by canterberrie** [comment]
@IamManchanda Taking a look now!

File: src/building-blocks/3-up-news-cards/3-up-news-cards.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!-- Font Awesome -->
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" rel="stylesheet"> 
+
+<!-- Use this stylesheet for typography in head of your HTML doc if you want :) -->
+<link href="http://fonts.googleapis.com/css?family=Roboto:700,400" rel="stylesheet">
```
Comment (canterberrie) [inline src/building-blocks/3-up-news-cards/3-up-news-cards.html]: It's best not to use custom fonts in Building Blocks in general. We want BBs to be pretty stripped of styling as much as possible.

File: src/building-blocks/3-up-news-cards/3-up-news-cards.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        background-color: inherit;
+        color: inherit;
+      }
+    }
+  }
+}
+
+.news-card {
+  background-color: $white;
+  font-family: Roboto;
```
Comment (canterberrie) [inline src/building-blocks/3-up-news-cards/3-up-news-cards.scss]: Remove font-family

Comment (canterberrie) [inline src/building-blocks/3-up-news-cards/3-up-news-cards.scss]: or we can tie it into a settings variable like $body-font-family

**Comment by IamManchanda** [comment]
@canterberrie .... done is this good now?

File: src/building-blocks/3-up-news-cards/3-up-news-cards.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<!-- Start of news card component below -->
+
+  <div class="row">
+    <div class="small-12 medium-4 columns">
```
Comment (canterberrie) [inline src/building-blocks/3-up-news-cards/3-up-news-cards.html]: Rows & columns should be in a separate layout.html file. The layout file will be what shows up in the iframe preview on the Building Blocks site. The 3-up-news-cards.html should only contain markup relating to the Building Block itself.

Comment (rafibomb) [inline src/building-blocks/3-up-news-cards/3-up-news-cards.html]: for this one - I'll add it to demonstrate how it works.

Comment (IamManchanda) [inline src/building-blocks/3-up-news-cards/3-up-news-cards.html]: i tend to concur with @rafibomb .... This doesn't need layout file ... as this demo's the 3-up thing!

**Comment by rafibomb** [comment]
@IamManchanda I already merged this to master - I went back and added your twitter - Thanks!

**Comment by IamManchanda** [comment]
No issue @rafibomb

**Comment by rafibomb** [comment]
I made some edits to the block for class naming - so that the styles don't blead into other building blocks and created a layout.html

```
<div class="row small-up-1 medium-up-3">
  <div class="column">
    {{> news-card}}
  </div>
  <div class="column">
    {{> news-card}}
  </div>
  <div class="column">
    {{> news-card}}
  </div>
</div>
```
Great work!

**Pull request closed without merging**